scripts/setup.sh

```diff
@@ -12,20 +12,55 @@ abs_root_dir="$(realpath "$rel_root_dir")"
 # tdp-collection
 TDP_COLLECTION_URL=https://github.com/TOSIT-IO/tdp-collection
 TDP_ROLES_PATH="$abs_root_dir/ansible_roles/collections/ansible_collections/tosit/tdp"
-TDP_COLLECTION_COMMIT=7076c9c95a14f701e510d02520c5c4901a7ac2bd
+
```

Comment:
I renamed TDP_COLLECTION_COMMIT to be explicit that it was associated to the stable download